managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -1033,6 +1035,90 @@ public Result index(UUID customerUUID, UUID universeUUID) {
     return Results.status(OK, universe.toJson());
   }
 
+  public Result pause(UUID customerUUID, UUID universeUUID) {
+    Universe universe;
+    try {
+      universe = checkCallValid(customerUUID, universeUUID);
+    } catch (RuntimeException e) {
+      return ApiResponse.error(BAD_REQUEST, e.getMessage());
+    }
+    Customer customer = Customer.get(customerUUID);
+
+    LOG.info("Pause universe, customer uuid: {}, universe: {} [ {} ] ",
+            customerUUID, universe.name, universeUUID);
+
+    // Create the Commissioner task to pause the universe.
+    PauseUniverse.Params taskParams = new PauseUniverse.Params();
+    taskParams.universeUUID = universeUUID;
+    // There is no staleness of a pause request. Perform it even if the universe has changed.
+    taskParams.expectedUniverseVersion = -1;
+    taskParams.customerUUID = customerUUID;
+    // Submit the task to pause the universe.
+    TaskType taskType = TaskType.PauseUniverse;
+
+    UUID taskUUID = commissioner.submit(taskType, taskParams);
+    LOG.info("Submitted pause universe for " + universeUUID + ", task uuid = " + taskUUID);
+
+    // Add this task uuid to the user universe.
+    CustomerTask.create(customer,
+      universe.universeUUID,
+      taskUUID,
+      CustomerTask.TargetType.Universe,
+      CustomerTask.TaskType.Pause,
+      universe.name);
+
+    LOG.info("Paused universe " + universeUUID + " for customer [" + customer.name + "]");
+
+    ObjectNode response = Json.newObject();
+    response.put("taskUUID", taskUUID.toString());
+    Audit.createAuditEntry(ctx(), request(), taskUUID);
+    return ApiResponse.success(response);
+
+  }
+
+
+  public Result resume(UUID customerUUID, UUID universeUUID) {
+    Universe universe;
+    try {
+      universe = checkCallValid(customerUUID, universeUUID);
+    } catch (RuntimeException e) {
+      return ApiResponse.error(BAD_REQUEST, e.getMessage());
+    }
+    Customer customer = Customer.get(customerUUID);
+
+    LOG.info("Resume universe, customer uuid: {}, universe: {} [ {} ] ",
+            customerUUID, universe.name, universeUUID);
+
+    // Create the Commissioner task to resume the universe.
+    ResumeUniverse.Params taskParams = new ResumeUniverse.Params();
+    taskParams.universeUUID = universeUUID;
+    // There is no staleness of a delete request. Perform it even if the universe has changed.
```

Comment:
The comment is copy/pasted from destroy() but not updated. There is another request here, not delete.